operator/pkg/manifest/operations.go

```diff
@@ -187,7 +188,7 @@ func getManifestProcessor(deployInfo types.InstallInfo, logger logr.Logger) (typ
 
 func (o *Operations) consistencyCheck(deployInfo types.InstallInfo) (bool, error) {
 	// verify CRDs
-	if err := resource.CheckCRDs(deployInfo.Ctx, deployInfo.Crds, deployInfo.ClusterInfo.Client,
+	if err := resource.CheckCRDs(deployInfo.Ctx, deployInfo.Crds, o.renderSrc.ToClient(),
```

Comment:
IMO something called a render src should not have methods for clients. The interface is not clearly cut here.

Reply:
this implementation was changed.